examples/kafka/cifar10/cifar10_kafka.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<p>is 172.17.0.2 your local address or is it one that you select explicitly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1919/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
That is the default IP address Kind will be exposed on. Not sure if there are limitations where this is not the case for Kind.